### PR TITLE
EventBus Proxy

### PR DESCRIPTION
**Pull request by hendrikebbers**
Provides a proxy for the eventbus to provide injection before DP has been full booted.
Some small changes for integration tests

---
This change is [Reviewable](https://reviewable.io/reviews/canoo/dolphin-platform/555)

### CODE REVIEW

**Comment by hendrikebbers** [comment]
Fix for https://github.com/canoo/dolphin-platform/issues/554

File: integration-server/src/main/java/com/canoo/dolphin/integration/server/property/PropertyTestController.java [hunk 1]
```diff
@@ -114,6 +114,8 @@ public void addChangeListener() {
         shortChangedSubscription = model.shortValueProperty().onChanged(e -> model.setShortValueChanged(true));
         stringChangedSubscription = model.stringValueProperty().onChanged(e -> model.setStringValueChanged(true));
         uuidChangedSubscription = model.uuidValueProperty().onChanged(e -> model.setUuidValueChanged(true));
+
+        model.stringValueProperty().onChanged(e -> System.out.println("Value changed from " + e.getOldValue() + " to " + e.getNewValue()));
```
Comment (timo0) [inline integration-server/src/main/java/com/canoo/dolphin/integration/server/property/PropertyTestController.java]: sout?

File: integration-server/src/main/java/com/canoo/dolphin/integration/server/listener/TestClientSessionListener.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    public static boolean eventBusInjected = false;
+
+    @Override
+    public void sessionCreated(ClientSession clientSession) {
+        eventBusInjected = (eventBus != null);
+    }
+
+    @Override
+    public void sessionDestroyed(ClientSession clientSession) {
+
```
Comment (timo0) [inline integration-server/src/main/java/com/canoo/dolphin/integration/server/listener/TestClientSessionListener.java]: empty?

File: integration-server/src/main/java/com/canoo/dolphin/integration/server/property/PropertyTestController.java (hunk 1 above)
Comment (hendrikebbers) [inline integration-server/src/main/java/com/canoo/dolphin/integration/server/property/PropertyTestController.java]: We can remove this after the first travis build. Want to see how travis behaves here.

File: integration-server/src/main/java/com/canoo/dolphin/integration/server/listener/TestClientSessionListener.java (hunk 2 above)
Comment (hendrikebbers) [inline integration-server/src/main/java/com/canoo/dolphin/integration/server/listener/TestClientSessionListener.java]: yes, it's only import to have the class and inject an eventBus. By doing so we can check if the eventbus can be used if he was injected before DP bootstrap.

**Pull request merged**